pennylane_lightning/core/src/simulators/lightning_qubit/gates/tests/Test_GateImplementations_Nonparam.cpp

```diff
@@ -243,6 +242,32 @@ template <typename PrecisionT, class GateImplementation> void testApplyT() {
     }
 }
 PENNYLANE_RUN_TEST(T);
+
+template <typename PrecisionT, class GateImplementation> void testApplySX() {
+    // Test using |000> state
+    using ComplexT = std::complex<PrecisionT>;
+    const size_t num_qubits = 3;
+
+    ComplexT z(0.0, 0.0);
+    // ComplexT p(static_cast<PrecisionT>(0.5),static_cast<PrecisionT>(0.5));
+    // ComplexT m(static_cast<PrecisionT>(0.5),static_cast<PrecisionT>(-0.5));
```

Comment:
Do you intend to come to this later? Any questions? If not, I think you can delete this comment.